### ISSUE TITLE
Order of operations not preserved when using classes or switch statement

### ISSUE THREAD

**Issue by rOzzy1987** [coderaiser/minify#119]
***Using classes:***
Considering the following code which works fine on its own, minifier merges the variable declarations, and produces (a well deserved) error in runtime:

original code:

```js
var a = "foo";
var b = 123.4;
class C {
    f() {
    }
    constructor() {
        this.d = a;
        this.e = b;
    }
}
var g = new C();
globalThis.h = g;
globalThis.i = g;
```
minified code, formatted:

```js
var a='foo',
    b=123.4,
    g=new C();
class C{
  f(){}
  constructor(){
    this.d=a;
    this.e=b
  }
}
globalThis.h=g;
globalThis.i=g;
```

Notice that instantiation of 'c' jumped before the class declaration, thus throwing a runtime error.

***Using switch statement:***
The switch statement case is similar.

Fom this:

```js
var a = "x";
switch (a) {
    case "a":
        var b = "foo";
        console.log(b);
        break;
    case "x":
        var c = "bar";
        console.log(c);
        break;
}
```
To this:

```js
var a='x';
switch(a) {
  case 'a':
    var b='foo',
        c='bar';
    console.log(b);
    break;
  case 'x':
    console.log(c);
    break
}
```

tested with **minify v11.0.1**

**Comment by coderaiser** [comment]
It should be fixed in [`@putout/plugin-minify`](https://github.com/coderaiser/putout/tree/master/packages/plugin-minify#readme)

**Comment by coderaiser** [comment]
Just fixed in [`@putout/minify@3.8.0](https://github.com/putoutjs/minify/releases/tag/v3.8.0) 🎉. Please re-isntall **Minify**. Is it works for you?

**Comment by rOzzy1987** [comment]
Verified the fix, Thanks for the fast response!

Cheers mate!
